modules/core/src/lib/attribute.js

```diff
@@ -184,29 +184,35 @@ export default class Attribute extends BaseAttribute {
     return false;
   }
 
-  updateBuffer({numInstances, bufferLayout, data, props, context}) {
+  updateBuffer({numInstances, bufferLayout, data, startIndex, endIndex, props, context}) {
```

Comment:
With the "table" nomenclature I have been propagating for when it comes to `data` related props and names, these parameters could be called `firstRow`, `lastRow`.

I think that naming would make these parameter easier to understand at a glance, i.e. is this:
- Index into the original data array (data table)?
- index into the generated vertex arrays?
- Index into the indices array?

Calling them row would make it obvious that this is the first case.